### PR TITLE
Correction Ucopia #13031

### PR DESCRIPTION
**Pull request by mathieu-artic**
Ucopia doesn't use index, bad informations used by librenms
Correction of #13031
image

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by mathieu-artic** [comment]
Fail is normal, the index has been deleted

**Comment by PipoCanaja** [comment]
Hi @mathieu-artic 
Tests must be fixed in order to merge. So you have to generate again (using save-test-data) and commit the new file. That will allow the regression test to succeed. 
Thx

File: includes/definitions/discovery/ucopia.yaml [hunk 1]
```diff
@@ -12,9 +12,8 @@ modules:
                 -
                     oid: highAvailabilityStatus
                     value: highAvailabilityStatus
-                    num_oid: '.1.3.6.1.4.1.31218.3.7.0.{{ $index }}'
+                    num_oid: '.1.3.6.1.4.1.31218.3.7.0'
```
Comment (PipoCanaja) [inline includes/definitions/discovery/ucopia.yaml:15]: Hi @mathieu-artic 
Why don't you do `num_oid: '.1.3.6.1.4.1.31218.3.7.{{ $index }}`. Index will always be "0" (which is fine if only one highAvailabilityStatus is provided.

Comment (mathieu-artic) [reply includes/definitions/discovery/ucopia.yaml:15]: (plus simple en français)
pour une raison que je ne connais pas, LibreNMS n'a pas l'air d'interpréter de la même façon
on peut le voir sur le screenshot d'hier (#13031) il met "OK" au lieu de ce qui est indiqué dans le YAML
Pire, pour "Debug Value" ce n'est pas le bon chiffre qui est interprété et il affiche "Emergency" à la place de "Notice"
J'ai vérifié dans le débug, le nombre que LibreNMS reçoit est le bon

Comment (PipoCanaja) [reply includes/definitions/discovery/ucopia.yaml:15]: Plus simple pour nous, mais pas forcément pour les autres :) 

My suggestion is to remove the "0" : 
from 
`                    num_oid: '.1.3.6.1.4.1.31218.3.7.0.{{ $index }}'`
to
`                    num_oid: '.1.3.6.1.4.1.31218.3.7.{{ $index }}'`

Could you try this variant ?

Comment (mathieu-artic) [reply includes/definitions/discovery/ucopia.yaml:15]: Good idea ! I will try this

Comment (mathieu-artic) [reply includes/definitions/discovery/ucopia.yaml]: i'ts work great, and all tests passed, thank's !

**Review by PipoCanaja — APPROVED** [review]
LGTM

**Pull request merged**